### ISSUE TITLE
[Wiki] takes more than 4 seconds to complete loading

### ISSUE THREAD

**Issue by rassi0429** [Yellow-Dog-Man/resonite-wiki#30]
### Is your feature request related to a problem? Please describe.

Thank you for hosting the Wiki!
Currently, when accessing from Japan, it takes more than 4 seconds to complete loading. 
The Wiki appears to be hosted in Finland, which is physically far from Japan. there are times when it can't be helped.

![image](https://github.com/Yellow-Dog-Man/resonite-wiki/assets/91118218/63e9180d-ede0-4f2e-91b0-d0a5f38ad5c1)

### Describe the solution you'd like

Proxy Wiki with a CDN such as Cloudflare. Alternatively,they can proxy only static assets.
This improves load times by generating cache in CloudFlare data centers around the world.

### Describe alternatives you've considered

Basically CloudFlare should be fine to proxy all requests. It also offers SSL and DDoS protection, which should be a big advantage.
It provides HTTP3 and parallel load caching, so even if Apache doesn't support that feature, CloudFlare will upgrade the protocol to support it.
